README.md

```diff
@@ -61,6 +61,45 @@ yarn prisma:seed
 yarn dev
 ```
 
+### Common Issues
+
+#### error: server: unhandled promise rejection: [object Promise]: Error: connect ECONNREFUSED ::1:4551
+
+```bash
+info: 🚀 API listening at http://localhost:3333
+error: server: unhandled promise rejection: [object Promise]: Error: connect ECONNREFUSED ::1:4551
+error: server: unhandled promise rejection: [object Promise]: Error: connect ECONNREFUSED ::1:4551
+error: server: unhandled promise rejection: [object Promise]: Error: connect ECONNREFUSED ::1:4551
+```
+
+This error will show on startup after running `yarn dev`.
+
+What this means is the local **ngrok** service is not running.
+
+To start up **ngrok** locally, you need to run `COMPOSE_PROFILES=ngrok docker-compose up -d`.
```

Comment:
FYI: alternatively, you can run yarn run dev:services:all and we can remove stripe from that command in package.json for now.

Reply:
Right, I just mentioned this as the workaround now. Once we modify the scripts we can update this

Reply:
updating the script in this PR now: [https://github.com/maybe-finance/maybe/pull/93](https://github.com/maybe-finance/maybe/pull/93) 👍

Reply:
Wondering if we should update the setup commands now?

Reply:
I think this issue is now resolved with the latest changes to the commands in package.json

Reply:
Aye, docker compose should be set. Also since we already have NX, we can embed the env var requirements in that config. It will also be more flexible.